doc/Pacemaker_Explained/en-US/Ch-Advanced-Options.txt

```diff
@@ -235,36 +237,42 @@ positive and negative constraint. Eg.
 which has the same long-term consequences as discussed earlier.
 
 [[s-failure-migration]]
-=== Moving Resources Due to Failure ===
+=== Moving Resources Due to Repeated Failure ===
```

Comment:
migration-threshold=1 does not count as repeated which would make the original title correct.